cime_config/machines/config_machines.xml

```diff
@@ -3510,7 +3510,6 @@
         <arg name="latency_priority">--latency_priority $ENV{LTC_PRT}</arg>
         <arg name="stdio_mode">--stdio_mode prepended</arg>
         <arg name="thread_vars">$ENV{JSRUN_THREAD_VARS}</arg>
-        <arg name="smpiargs">--smpiargs="-gpu"</arg>
```

Comment:
You should not enable GPU aware MPI by default.

Reply:
yeah, that is why I remove this line

Reply:
Thanks, I understand. You still need to add this back for the GPU compilers right?

Reply:
ok